src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs

```diff
@@ -22,10 +22,11 @@ public InlineCommentPeekResult(InlineCommentPeekViewModel viewModel)
 
         public Action<IPeekResult, object, object> PostNavigationCallback => null;
 
-        public event EventHandler Disposed;
+        public event EventHandler Disposed = delegate { };
```

Comment:
What does the `= delegate { }` do here considering the `Disposed` event is being called?

Reply:
It stops CA complaining that `Disposed` isn't being used.

The alternatives are:
1. An attribute to ignore this error.
2. Explicitly defining `add { } and remove { }`.

This seemed like the least worst option. I'm open to alternatives.

Reply:
What I meant was that on line 29 we're calling `Disposed?.Invoke(this, EventArgs.Empty);`, so `Disposed` _is_ being used. Will `Disposed` even work with the event defined like this?